src/apply.jl

```diff
@@ -83,7 +83,7 @@ function apply(table, t::Transform; cols=_get_cols(table), header=nothing, kwarg
     # Extract a columns iterator that we should be able to use to mutate the data.
     # NOTE: Mutation is not guaranteed for all table types, but it avoid copying the data
     coltable = Tables.columntable(table)
-    # Combine the Vector{Vector} of components into a Matrix
+    # Consolidate the Vector{Vector} components into a single array, which might be a Vector.
```

Comment:
I commented on this in https://github.com/invenia/FeatureTransforms.jl/pull/80#discussion_r614855184 since it's relevant to my other comments there